bin/varnishd/cache/cache_req_fsm.c

```diff
@@ -141,7 +141,7 @@ Resp_Setup_Deliver(struct req *req)
 	http_PrintfHeader(h, "Age: %.0f",
 	    floor(fmax(0., req->t_prev - oc->t_origin)));
 
-	http_SetHeader(h, "Via: 1.1 varnish (Varnish/6.4)");
+	http_SetHeader(h, "Via: 1.1 varnish (Varnish/6.5)");
```

Comment:
Isn't it updated here?

Reply:
My mistake, sorry. Merge and we are good.